internal/util/test/webhooks.go

```diff
@@ -4,110 +4,13 @@ import (
-hIXG+ywiIgqUhZ6uwGR2IN5moTGG58PIl1FHZzTu9DjZ6vTlrswDkEln1ELoWOwR
-EKrjRRTd5dF6IWGyweuFvmQgUOtwF6JLSFz3jnUZfUc8sJwgnMmRGorjnWA2KLOe
-AErSwlTO7nJ79Xv+t7coXc5pg+d1NCsEjowsQe66QNLxQlRNQZ3Bw6bqSUOuY9nP
-WxYFeHoUN4yW+LmdoJRDRWgaio6GbbtbgrLhGx9LyMtu51667Bbt9DRR7e7FVOPj
-gkd0/2nDifkCjsoAWGl6RKJFym1yOQIDAQABAoICAAIObyTuNsneVBOY7A1VFd4X
-d+EV8+TYDr5KusYCXRA0ySBz2EaXbOoS1wiyGbFyZSnvVS+w76DX2KPvLdngC93D
-wT+vlfp4B+PXtlzs4Um/3ZHOCo3Y/lEs8EIRyqZgcjdHUOEDLNOLI7nH54B+kMnd
-xXxM/0Zn0qVITV5EmRNi+auNYe0cO5Ezjl0HX2D14IxjfT3gywreis9zjZVGqHNu
-nqFTchOAd/8m+C4ZTvECtDPuCx5ds02PyFls+uT680KW6cVmm0+JfI8W/Z9iZ6cn
-EJYl9S1frAKgCrzBVcjSRcMEA3nlOWm0mHm/3u1lcnnkNgbiBkui0z5wQfXUJ4rN
-SgODZTCUGlR50McmDjcfKWX0chRSu5ApNdt/6N8hqrj7Ao/Y7lANl+RH7WzgcA8I
-Wf/3fRBuIFLlfB2PkiH7rioi7VVioCRQfLq7g4cZZE8fZj377kayL9ilr079CDbD
-7bhhfdNUNcgi/EHxT0ZcrK8QakgILqVPeFcVvr6Nlu2KuTrjdDbvzoSeqFGs4x06
-PYgwp31Nscfm/+wKWtvX95poPUf1zcV3iCB/K8BSQ2LB2cijDbXIkRHMWx2eEhXA
-Dx95RCvMep4AXJNcTqkuRI2/PQNbdur1uMl6DtEp8P5otaNGEs4mUjZoNXIfSt6y
-VFUpSfaW5hyvxZIyMcttAoIBAQD1AWr9UFc8hDLV0z+Xyi0YUDU93b2TVTdN2wNk
-Ar8EFfRQbUNvgbPLQTRYXy9xMM1Lh+/5atov7DRxbxycS3pZPV5x8aL2DHoYnwhl
-r3cEZOKjWIytfR4Wl2kekidV7hblLBu3inqWP+8b2yBD1T49q9XDy2JEdsbVGA52
-iAPIHUdg9G5Qye5XpZ8pbyhWCTD2rJoYZqPMLoTuIXsk8p1OqdHNdUZFLtDOXCp/
-OsookB+3r5nkKF1tesmGWnsQKHlFh2nMlNbbto9Eq41q+wrqGAtz+e5PWBEYz3c3
-O+NIsJZjAfijiFw5BbCzIGYqjSuV8mk/hlU+pEP9Qq0BDJDTAoIBAQDHLNsYqNQn
-e4V7Jmd3bAG5jLyVzpiZ9wKKoFAV6+cSX1UTgbB0m+OQDEo4kXr7s9Qlb1YonZdN
-NZ+JaR31E9WWE1XV560lptlKv2WHWXzQ/77nYJqi4FunKIpsgAyOwum2PbK0W3LR
-G1mGgt7lLXzStEpklFFo41l+q4oq7rWDY9JcQA7w8KmS+NbyCYfzEgCw48e5tt01
-AhXv648KQ1iArlMJnvklw29P2Z4ZjNd12u65kzMf0jt/qGVFAz6pmnNs0NeT1Lry
-VtSZVLsmz2nMJxv9ynmvEsDK3u2bBz4yu06fTSZXC3HA2fOnA3PzR8b0A+LznBAs
-tRFjKUtiL2lDAoIBAE2Si1t06ooAmU/WONZIbwq5qoHSCUnyuzXtWB98NxqGEPq9
-/ZH6kQCTvo0UZiMCfp2hhruDt11g/iHEOlzKEQzcP2D4Ts50mNveznvTmG1Nu+xY
-BwKPEM26VtAVGls8rQcWdhggwjt8NqqtMIQJqlFAbMg3Rv/CU6X4neutmPTtQEJU
-YfE2Xj5l9ATcavsCofwYHhoHtWjaecAj3aePIXtcjU7uCLey4O3VhKcP6B37s+8m
-rCTvGtWlANWHJFuzVOJMw5TAs16RaL8nSflLTuNbhZTM10VL7u3aEjbswUOslZD3
-eM/eRsTPJmkwouhqFhi9zdENKURSIiy3xasFXX0CggEBAIMhEgITLNHtrbydVSM3
-lY8ejA4o3SkIicDQuMkl7ZSX9QAJsy2icFim7rp9dTV/eF1JBrVow3MOqcEi1plC
-fKz4f9t7UvBl/8sJZYONs/I5XeshG8293jmYJEI4r4vR4WKdDilVx6rJ0dAQG0VR
-BEwIbK35Q/vFKmeN8kI/tHsTXixj9DJGj632bDUfd0TdhlzvKdAoB0zd2amCMUM1
-gW/+1SaRZkCpgYBVqpPxuOybve2pbtA1bfym1j1wXHH3AKUUfcmTFZ108zUbprdv
-eJzy5qfZBPHxa8JksJQPWbC2xpV0iphmLXclRxee21jA2chPQrzV84QrlY3uYvX6
-dtcCggEBAM4Y0hHv677X1ivibdgeJfCNCx69WkKwqmwxDkR6PYEKHMJu9HQ1ttNN
-4vZiKl2cFhIb7cLyAkWzLzhDr5NlzeHJButG0XTNFFHwFoEtMGDpQKffZ6GtTsLB
-KN+kyxGgdrVLP55mhoWe42tI7TkwSnl3o5RgpX+b2pAcZ8s/F0jqm9E2Ub+tNp2Z
-aTT92eGFDCftg0FohOm6LX2bteuhREbVIycWy/I67kJ/M2F/70gCEdBw2LJsiGyO
-FK4IYsWdsJCKft+/IlUSgYYWY6GHOcfP3R34ibx7P244kqwknmTzxBKYnybGeyVT
-ctFsgXhf5+tDgbBZpcuTMpd3KnaDUYg=
------END PRIVATE KEY-----`
+//nolint:revive
+var KongSystemServiceCert, KongSystemServiceKey = certificate.MustGenerateSelfSignedCertPEMFormat(
```

Comment:
As I can see this cert is used only in `ensureAdmissionRegistration`, could we just move its creation in there?

Reply:
Not only, it's used in
- `test/conformance/gateway_conformance_test.go`
- `test/integration/suite_test.go`
- `test/integration/webhook_test.go` ( mentioned `ensureAdmissionRegistration`)

Reply:
Ah, sorry 🤦 Nevermind.